### ISSUE TITLE
location of local timeline

### ISSUE THREAD

**Issue by arkahys** [mastodon/mastodon#5784]
I don't understand attributs filter for local timeline.
I'am in guadeloupe i read post from swiss, and others distant contry.

there is a way to select region or setup location and a range option ?
And a language timeline ?

**Comment by Cassolotl** [comment]
Yes, you can choose the site language and use language filters in Settings:

![screen shot 2017-11-22 at 12 31 13](https://user-images.githubusercontent.com/6607512/33127635-2c790d68-cf81-11e7-9c88-eea76e4f35fc.png)

![screen shot 2017-11-22 at 12 32 25](https://user-images.githubusercontent.com/6607512/33127648-3bd8136c-cf81-11e7-8ea5-4a1eca9ea5fb.png)

Does that help?

**Comment by unarist** [comment]
from [our FAQ](https://github.com/tootsuite/documentation/blob/master/Using-Mastodon/FAQ.md#i-dont-get-the-local-vs-federated-timeline-thing):

> **“Local” toots are from the server you signed up at.** “Federated” are the all the toots your server knows about. (It's complicated, but short version is: “toots from people that you and other locals follow.”)

**Comment by Cassolotl** [comment]
Handy diagram of that:

![local and federated timelines](https://user-images.githubusercontent.com/6607512/33132404-8321b894-cf91-11e7-8bfd-b43b12d0c93c.png)

**Comment by Vince42** [comment]
I prefer to use all applications with an English interface, but I would like to receive local news in my native language. It would be nice, if you could split both aspects. 😄

**Comment by Cassolotl** [comment]
@Vince42 The site's language and the language filter for the public timelines are separate, so that should be possible!

**Comment by Vince42** [comment]
@Cassolotl Yes, I know. Maybe the term "local timeline" is misleading then. I would expect to see in "local timeline" all toots that originate from users in my region, no matter which language they are written in; otherwise the timeline should be named differently. My understanding is, that filtering languages does just filter languages, which in turn means that people writing in English and sitting in Japan would not be seen by a Japanese user, right?

**Comment by nightpool** [comment]
local means "local to your server" not "local to your geographic location"

**Comment by Cassolotl** [comment]
@Vince42 

>Maybe the term "local timeline" is misleading then.

I definitely agree with this. It's intuitive for people who play with computers and servers a lot, where "local" means something quite specific and technical - but for people who are maybe just looking for a Twitter alternative and who are not super computer-minded but just want a little light social media, "local" is a geographic term that needs explaining by established members, often with diagrams like the one I posted above. Diagrams should not be necessary, and I'm not sure how but I think we can do better.

This is something that comes up fairly often, from what I've seen so far - "local timeline" and "federated timeline" are terms that are just jargon/gibberish to people new to the fediverse. Some people have tried to correct it by renaming those timelines, like "planet" and "federation", I think?

I think for a new type of social media with a bunch of unfamiliar features there can sometimes be some confusion for new users that can be made a little easier by established members explaining stuff, and to an extent that's to be expected, but I also think that there could be better names for the public timelines that would make things easier for newbies.

So yeah, basically, I very much agree. I picked it up fast because I had played with GNU Social a year or two before I joined Mastodon, but I think for most people it's needlessly obscure.

>My understanding is, that filtering languages does just filter languages, which in turn means that people writing in English and sitting in Japan would not be seen by a Japanese user, right?

Yes, that's right! If someone in Japan is using Mastodon and looking at their local timeline with the language filters set to only show posts in English, they will not see geographically local toots from Japan in Japanese made by people sharing their instance - but they will see English toots from anywhere in the world that were posted by other users from their instance.
